Source/drlg_l2.cpp

```diff
@@ -528,7 +528,7 @@ void LoadPreL2Dungeon(char *sFileName, int vx, int vy)
 		for (i = 0; i < rw; i++) {
 			if (*lm != 0) {
 				dungeon[i][j] = *lm;
-				dflags[i][j] |= DFLAG_EXPLORED;
+				dflags[i][j] |= 0x80;

```

Comment:
```suggestion
				dflags[i][j] |= BFLAG_EXPLORED;
```

Reply:
This is `dflags` not `dFlags` at the moment we don't know it's exact meaning.

Reply:
lol! :D